### DISCUSSION TITLE
Browsershot not taking multi-languages screenshot properly.

### DISCUSSION THREAD

**Discussion by rohit-codaemon** [spatie/browsershot#765] [Bugs]
I have installed the "spatie/browsershot": "^3.57", in laravel 8.
When i am taking the screenshot of the HTML generated dynamically with other languages. It is showing only empty boxes.
I have checked the HTML is having proper text of the language text. I have tried languages like Arabic, Japanese, Chisese, Korean, Hindi and Marathi but the issue persist on all.
any help will be appriciated. Thank you

**Comment by jay-r-7span** [comment]
> I have installed the "spatie/browsershot": "^3.57", in laravel 8. When i am taking the screenshot of the HTML generated dynamically with other languages. It is showing only empty boxes. I have checked the HTML is having proper text of the language text. I have tried languages like Arabic, Japanese, Chisese, Korean, Hindi and Marathi but the issue persist on all. any help will be appriciated. Thank you

I am getting the same issue. Did you get any solution bro?

**Comment by bessone** [comment]
I noticed the problem with an Arabic site today, I found an issue of Puppeteer that talks about similar problems, still open

https://github.com/puppeteer/puppeteer/issues/4424

In another issue there is talk of installing the missing fonts, I haven't yet tried if it works.

https://github.com/puppeteer/puppeteer/issues/4996
